engine/api/mailer/Mailer.php

```diff
@@ -92,4 +107,13 @@ static function createActivationEmail(string $address, string $token)
         $content = "Voici le lien pour activer ton compte: $base_url/activation?token=$token";
         return Mailer::create()->createEmail($address, $subject, $content);
     }
+
+    static function createEventPublicationEmail(Event $event)
+    {
+        $base_url = env("BASE_URL");
+        $subject = "Nouvel événement sur l'intranose";
+        $content = "Un nouvel événement a été publié sur l'intranose ! Pour aller le voir : <a href = '$base_url/evenements/$event->id' >Nouvel événement</a>";
```

Comment:
Maybe include the name of the event somewhere? You could also add the deadline?

Reply:
Good one, I just added some more informations there 🤝